plugins/env/app/models/environment_variable.rb

```diff
@@ -0,0 +1,16 @@
+class EnvironmentVariable < ActiveRecord::Base
+  belongs_to :parent, polymorphic: true
+  belongs_to :deploy_group
```

Comment:
Does this mean that if I want to have AIRBRAKE_API_KEY defined, I would need to manually add the same value to each deploy group?  (pod98-101, pod1-6, gamma1-6)